cloudbuild.yaml

```
@@ -3,6 +3,6 @@ options:
   substitution_option: ALLOW_LOOSE
   machineType: E2_HIGHCPU_32
 steps:
-- name: gcr.io/k8s-testimages/krte:latest-master
+- name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud
```

```suggestion
- name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20240718-5ef92b5c36
```

Maybe stick to a tag instead of latest ?